docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  backend:
+    build: ./backend
+    ports:
+      - "127.0.0.1:8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      mongodb:
```

Не увидел сразу. Тут тоже надо поменять:
```suggestion
      db:
```